person.rb

```diff
@@ -9,13 +9,13 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @parent_permission = parent_permission
   end
 
```

Comment:
**[Optional]**

- [ ] Kindly include the **Future Feature** section in your readme file to make it descriptive and professional :blush:

Reply:
thanks